requirements.txt

```diff
@@ -2,4 +2,9 @@
 Jinja2==2.11.2
 MarkupSafe==1.1.1
 Werkzeug==1.0.1
-python-dotenv==0.15.0
\ No newline at end of file
+python-dotenv==0.15.0
+flasgger==0.9.5
+py==1.10.0
+PyYAML==5.3.1
+pytest==6.2.1
```

Comment:
`pytest` and `pytest-mock` are test-spefic requirements and, actually, not needed I just want to run the application. There can be 2 requirements files: `requirements.txt` and `requirements-test.txt` for test-specific requirements

Reply:
fixed